tensorflow/python/feature_column/sequence_feature_column_test.py

```diff
@@ -466,9 +466,8 @@ def test_sequence_length_not_equal(self):
 
     with self.assertRaisesRegexp(
         errors.InvalidArgumentError, r'Condition x == y did not hold.*'):
-      _, sequence_length = sequence_input_layer({
-          'aaa': sparse_input_a, 'bbb': sparse_input_b})
-      self.evaluate(sequence_length)
+        _, sequence_length = sequence_input_layer({'aaa': sparse_input_a, 'bbb': sparse_input_b})
+        self.evaluate(sequence_length)
```

Comment:
This seems like the wrong indenting-- revert this one?

Reply:
Thanks @karmel. Updated.